app/src/main/java/ch/epfl/sweng/swenggolf/profile/User.java

```diff
@@ -24,7 +24,7 @@
     private String photo;
     private String preference;
     private String description;
-    private int points;
+    private int points = 0;
```

Comment:
The points are initialized twice, either we keep the "DEFAULT_POINTS=0" either we keep your "points = 0".
You can choose!

Reply:
You are absolutely right, I didn't pay attention to that...
I will correct it right away!

Reply:
The `DEFAULT_POINTS = 0` solution seems cleaner to me and more consistent with the general coding style of the app